Development-Guide/CAN-Filter-Guide/README.adoc

```diff
@@ -0,0 +1,60 @@
+== CAN Message Filter Guide
+----
+This is a basic guide to making a CAN message filter
+----
+
+==== Introduction
+The two main components of a filter are an ID and a Mask.
+The ID will be the unique identifier for the message you are looking for, and the Mask will determine which bits of the ID are important to sift through (this makes the proccess much more efficient and flexible). 
+
+IMPORTANT: A Mask must be able to adapt to future additions to a CAN network. It must be made *as tight as possible for longevity*.
+
+There is no set order to follow the steps, but to get started follow the steps below until you are comfortable.
+Another tip is to convert the hex numbers into binary numbers, determine the ID and Mask, and then convert those back into hex.
+
+.*Steps*
+. Determine the ID
+.. The ID will be the unique identifier for the message(s) that you are trying to receive
+.. The ID should be very specific and contain the important bits that will be compared to the incoming message. 
+The greater the variery of addresses, the more flexible your ID will be to accomodate the the variety
+.. For example: If you are looking for a message from a system that sends out messages with address 0b0000_0000_1111(0x00F) then you want your ID to look like 0b0000_0000_1111(0x00F) because there is only one type of address you are getting
```

Comment:
Use just 1 bytes for your example, having 12 binary places is kinda weird. Usually it'd be 8 or 16.